pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1720,6 +1726,41 @@ private void sendSegmentRefreshMessage(String offlineTableName, String segmentNa
     }
   }
 
+  private void sendUpdateQueryQuotaMessage(TableConfig tableConfig) {
+    QuotaConfig quotaConfig = tableConfig.getQuotaConfig();
```

Comment:
If the original table config had query quota but it has been removed, don't you want to send the message?

Reply:
Good idea, although it's very rare the query quota got removed once added.
Updated the PR.